### DISCUSSION TITLE
Configure Terminal Color

### DISCUSSION THREAD

**Discussion by laurent-n** [kingToolbox/WindTerm#667] [Ideas]
In kitty I used to configure background color of the terminal in differrent colors depending on the system class. for instance, dark red on critical production systems, 
It would be great if I can do the same here.

**Comment by kingToolbox** [comment]
Of course you can. You can modify `WindTerm_Profiles_Path/global/themes/dige-black/scheme.theme` to set any color and font you want.  

`WindTerm` are very customizable. You can read [this comment](https://github.com/kingToolbox/WindTerm/issues/57#issuecomment-790874266) to learn more. Subsequent versions will add many user interfaces to facilitate user customization.

**Comment by laurent-n** [comment]
The suggestion here is the ability to do it at session level, like it can be done for session tab color.

**Comment by kingToolbox** [comment]
I see, that's why I added the `session tab color` or `session color`. But I have temporarily abandoned the plan to use the `session color` as the background color, because many times the color of the font will be difficult to distinguish. It is difficult to design a set of colors that can be applied to multiple background colors. The use of background images is same.

I also have a design that uses the session color as the color of the view frame, so that it does not affect the display of the text, and it is quite eye-catching too. What do you think?

Let me know if you have any ideas, I would love to get feedback on this. :smile:

**Comment by laurent-n** [comment]
Hello,
I use only very dark color like that 
![image](https://user-images.githubusercontent.com/11587919/163833627-f267c19a-05a2-46c6-9639-2aa476849f3f.png)
or
![image](https://user-images.githubusercontent.com/11587919/163833831-993fcab5-fdaa-4190-9cec-c59e4611ab94.png)
it work fine with default colors, it's near enought of black to work fine in most of case, but far enought to avoid mitakes.
Colored frame could be a good idea too, but I like the idea to have frame as thin as possible, even invisible ...

**Comment by kingToolbox** [comment]
Ok, I will implement a prototype in `WindTerm_2.5.0_Prerelease_1` or `WindTerm_2.5.0_Prerelease_2`, please try it out and give your feedback then. Since `WindTerm` colorizes more text through lexer, I hope everything will be fine by then.

If there is any progress, I will update here.

**Comment by securitasis** [comment]
I'm new to WinTerm and I am using version 2.6.1 stable. I was looking for this exact same capability to colorize the background on my production Ubuntu systems so I know I am in prod and I'm more cautious to not make a mistake and break something. Was the option ever added to set a non-global customized background color in a saved session?
